### ISSUE TITLE
add to() to Measurement

### ISSUE THREAD

**Issue by mpound** [mpound/pdrtpy#62]
utils.to() is a bit cumbersome to use.  adding the functionality to Measurement makes sense. 
measurement2 = measurement1.to(unit)
where unit is either astropy Unit or str

**Comment by NanoExplorer** [comment]
I've been using `measurement2 = measurement1.convert_unit_to(unit)`, which is still a bit verbose but perhaps better than using the utils package.

**Comment by mpound** [comment]
yeah, i have sometimes used that. makes more sense to use NDData native method.
